framework/src/systems/AuxiliarySystem.C

```diff
@@ -927,6 +927,38 @@ AuxiliarySystem::computeNodalVarsHelper(const MooseObjectWarehouse<AuxKernelType
   }
 }
 
+void
+AuxiliarySystem::variableWiseRelativeSolutionDifferenceNorm(
+    std::vector<Number> & rel_diff_norms) const
+{
+  rel_diff_norms.resize(nVariables(), 0);
+  // Get dof map from system
+  const auto & dof_map = _sys.get_dof_map();
+
+  for (const auto & n : make_range(nVariables()))
```

Comment:
```suggestion
  for (const auto n : make_range(nVariables()))
```
no need to take a reference for something as small as a uint